tools/wallet/src/core/wallet_library.rs

```diff
@@ -151,3 +149,88 @@ impl WalletLibrary {
         &self.key_factory
     }
 }
+
+#[cfg(test)]
```

Comment:
same comment as elsewhere, this doesn't need to be a `mod`

Reply:
Updated